src/style.css

```diff
@@ -2,0 +1,144 @@
+.stars {
+  margin: 0;
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both, to avoid potential margin collapse. In this case, you have set `margin: 0;` which is acceptable for zeroing out all margins, but be mindful of this rule when setting individual margins.